### PR TITLE
Switch to uwsgi to run apps instead of gunicorn

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
@miltontony & @Rizziepit ready for review.

**Comment by Rizziepit** [comment]
+1

File: unicoremc/constants.py [hunk 1]
```diff
@@ -3,9 +3,13 @@
     '%(app_type)s-%(country)s%(suffix)s')
 
 SPRINGBOARD_MARATHON_CMD = (
-    "/var/praekelt/springboard-python/bin/gunicorn "
-    "--bind $HOST:$PORT "
-    "--paste %(config_path)s --preload"
+    "/var/praekelt/springboard-python/bin/uwsgi "
+    "--virtualenv /var/praekelt/springboard-python "
+    "--ini-paste %(config_path)s "
+    "--socket $HOST:$PORT "
```
Comment (miltontony) [inline unicoremc/constants.py]: This should be `--http`

**Comment by miltontony** [comment]
@smn other than that :+1:

File: unicoremc/constants.py (hunk 1 above)
Comment (smn) [inline unicoremc/constants.py]: Tried that earlier but:

```
uwsgi --virtualenv ./ve --ini-paste ./development.ini --socket localhost:8300 --processes 1 --threads 1 --http
[uWSGI] getting INI configuration from ./development.ini
*** WARNING: Can't find section "uwsgi" in INI configuration file ./development.ini ***
uwsgi: option `--http' requires an argument
getopt_long() error
```

and 
`--protocol http` doesn't and that seems to work.

Comment (smn) [inline unicoremc/constants.py]: A replacing socket with http works.

**Pull request merged**